src/test/java/com/microsoft/sqlserver/testframework/util/Util.java

```diff
@@ -301,6 +301,25 @@ public static boolean supportJDBC43(Connection con) throws SQLException {
         return (meta.getJDBCMajorVersion() >= 4 && meta.getJDBCMinorVersion() >= 3);
     }
 
+    /**
+     * Checks if object SYS.SENSITIVITY_CLASSIFICATIONS exists in SQL Server
+     * 
+     * @param Statement
+     * @return boolean
+     */
+    public static boolean serverSupportsDataClassification(Statement stmt) {
+        try {
+            stmt.execute("SELECT * FROM SYS.SENSITIVITY_CLASSIFICATIONS");
+        }
+        catch (SQLException e) {
+            // Check for Error 208: Invalid Object Name
+            if (e.getErrorCode() == 208) {
+                return false;
+            }
```

Comment:
what happens if this fails and it's not error 208?

Reply:
It would return true

Reply:
right, missed that ;)